amt/api/routes/projects.py

```diff
@@ -36,15 +69,19 @@ async def get_root(
         "projects": projects,
         "next": next,
         "limit": limit,
+        "start": skip,
         "search": search,
+        "lifecycles": get_lifecycles(request),
+        "publication_categories": get_publication_categories(request),
+        "filters": localized_filters,
     }
 
-    if request.state.htmx:
-        return templates.TemplateResponse(
-            request, "projects/_list.html.j2", {"projects": projects, "next": next, "search": search, "limit": limit}
-        )
-
-    return templates.TemplateResponse(request, "projects/index.html.j2", context)
+    if request.state.htmx and drop_filters:
+        return templates.TemplateResponse(request, "parts/project_search.html.j2", context)
+    elif request.state.htmx:
+        return templates.TemplateResponse(request, "parts/filter_list.html.j2", context)
+    else:
+        return templates.TemplateResponse(request, "projects/index.html.j2", context)
```

Comment:
is there a reason to move this into the else?

Reply:
Not particularly, but it reads more fluently IMO.